api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -17,6 +17,7 @@
 import logging
 from pathlib import Path
 from typing import Type
+import requests
```

Comment:
`requests` are good but it needs to be mentioned in our dependencies list in `setup.py`. In some environment latest versions of that package might be not available from python index repos (at least for the initial time: it should be noted by devops team).

So please:
- add that dependency as `requests>=2.14.2` (since epel-7 provides it as rpm) into `setup.py`

Reply:
Done